pages/file-dropzone/container.page.tsx

```diff
@@ -0,0 +1,120 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React, { useContext, useState } from 'react';
+
+import { Box, Button, Checkbox, Container, Header, SpaceBetween, Table } from '~components';
+import FileDropzone, { useFilesDragging } from '~components/file-dropzone';
+
+import AppContext, { AppContextType } from '../app/app-context';
+
+type DemoContext = React.Context<
+  AppContextType<{
+    onlyVisibleOnDrag: boolean;
+  }>
+>;
+
+export default function FileDropzoneContainer() {
+  const [files, setFiles] = useState<File[]>([]);
+
+  const [selectedItems, setSelectedItems] = useState<any>([]);
+  const { urlParams, setUrlParams } = useContext(AppContext as DemoContext);
+  const { onlyVisibleOnDrag } = urlParams;
+
+  const handleFilesChange = (newFiles: File[]) => {
+    const newValue = [...files, ...newFiles];
+    setFiles(newValue);
+  };
+
+  const removeFiles = () => {
+    const newValue = files.filter(file => !selectedItems.map((item: any) => item.name).includes(file.name));
```

Comment:
The files names might not be unique. It is better to compare the files by reference. If you pass files as table items you can just do includes(). Otherwise, you can include file as a property of an item and do `files.filter(file => !selectedItems.some(item => item.file === file))`